adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -151,9 +160,29 @@ private void validateAcquireTokenRequest(final AuthenticationRequest authenticat
                     ADALError.DEVELOPER_AUTHORITY_IS_NOT_VALID_URL);
         }
 
+        if (mTokenCacheAccessor != null) {
+            mTokenCacheAccessor.setAuthRequest(authenticationRequest);
```

Comment:
>mTokenCacheAccessor [](start = 12, length = 19)

can you set the telemetry request id in the constructor where we create the token cache accessor? Request id remains the same per AcquireTokenRequest, right?